connectivity/drivers/emac/TARGET_STM/TARGET_STM32H7/CMakeLists.txt

```diff
@@ -9,6 +9,8 @@ elseif("NUCLEO_H743ZI" IN_LIST MBED_TARGET_LABELS)
     add_subdirectory(TARGET_NUCLEO_H743ZI)
 elseif("NUCLEO_H743ZI2" IN_LIST MBED_TARGET_LABELS)
     add_subdirectory(TARGET_NUCLEO_H743ZI2)
+elseif("NUCLEO_H753ZI" IN_LIST MBED_TARGET_LABELS)
+    add_subdirectory(TARGET_NUCLEO_H743ZI2)
```

Comment:
OK with CL2, but this will not work with CLI1 :-(
I am afraid you should duplicate code...

Reply:
Yikes